Kitodo/src/test/java/org/kitodo/selenium/SearchingST.java

```diff
@@ -53,18 +71,31 @@ public void logout() throws Exception {
 
     @Test
     public void searchForProcesses() throws Exception {
+
         desktopPage.searchInSearchField("process");
         int numberOfResults = searchResultPage.getNumberOfResults();
         assertEquals("There should be two processes found",2,numberOfResults);
 
-        searchResultPage.searchInSearchField("proc");
-        numberOfResults = searchResultPage.getNumberOfResults();
-        assertEquals("There should be two processes found",2,numberOfResults);
+        searchResultPage.searchInSearchField("Second");
+        Thread.sleep(2000);
+        assertEquals("There should be two processes found",2,searchResultPage.getNumberOfResults());
 
-        //TODO: selenium is too fast here and counts results from previous search
         searchResultPage.searchInSearchField("möhö");
-        numberOfResults = searchResultPage.getNumberOfResults();
-        //assertEquals("There should be no process found",0,numberOfResults);
+        Thread.sleep(2000);
```

Comment:
You can use here more elegant method from Browser class: getDelayMaxAfterLinkClick()